app/helpers/application_helper.rb

```diff
@@ -136,7 +136,7 @@ def redirect_back_or_to(options = {}, response_status = {})
 
   def concurrent_events(event)
     return nil unless event.scheduled? && event.program.selected_event_schedules
-    event_schedule = event.program.selected_event_schedules.find_by(event: event)
+    event_schedule = event.program.selected_event_schedules.find { |es| es.event == event }
```

Comment:
Because of the changes I've made to Program#selected_event_schedules, it no longer returns an ActiveRecord::Relation but an Array, and arrays don't have `find_by` they just have `find`